backend/union/union.go

```diff
@@ -567,7 +567,7 @@ func (f *Fs) About(ctx context.Context) (*fs.Usage, error) {
 // This should return ErrDirNotFound if the directory isn't
 // found.
 func (f *Fs) List(ctx context.Context, dir string) (entries fs.DirEntries, err error) {
-	entriess := make([][]upstream.Entry, len(f.upstreams))
+	entriesList := make([][]upstream.Entry, len(f.upstreams))
```

Comment:
I'm happy to drop this (and a similar change) if requested.

It's a lot easier to understand a code block if it does not have two variables in a scope that differ by only a single letter.

Reply:
I think `entriess` is taking it a bit far... so this change is fine!